packages/7guis-counter/tsconfig.json

```diff
@@ -0,0 +1,16 @@
+{
```

Comment:
Do we want to have one `tsconfig.json` at the root of the repo and just have the rest of them be symlinks to the one at the root? Would be easy to make changes across all of them. Only question is if that works with Stackblitz. Otherwise just a single one at the repo root may be easiest to maintain. Thoughts?

Reply:
I feel like we shouldn't be symlinking anything committed. For stackblitz compatibility.